### PR TITLE
Support to Add Funds When Event is Over

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/3982

https://user-images.githubusercontent.com/12435965/129805335-3f8ab257-b6de-4171-b832-6b4876b1b574.mp4

### CODE REVIEW

File: components/AddFundsBtn.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { FormattedMessage } from 'react-intl';
 import AddFundsModal from './host-dashboard/AddFundsModal';
 import StyledButton from './StyledButton';
 
-const AddFundsBtn = ({ children, collective, host }) => {
+const AddFundsBtn = ({ children, collective, host, isFromCollective }) => {
```
Comment (Betree) [inline components/AddFundsBtn.js:8]: While it's correct to show "Add funds from Collective" in the interface, from the technical point of view we're really adding funds from the **parent** (ie. we could allow organizations to host events in the future). So I think we should rename this `isFromCollective` in the code, to rather have something like `isFromParent` or `isFromParentAccount`.

**Review by Betree — COMMENTED** [review]
Looks good to me, some feedback about variables naming

File: components/AddFundsBtn.js (hunk 1 above)
Comment (SudharakaP) [reply components/AddFundsBtn.js]: That makes sense. 👍🏽 I've renamed it to `isFromParent`. Let me know if everything looks good now.

File: components/collective-navbar/menu.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ export const NAVBAR_ACTION_TYPE = {
   APPLY: 'hasApply',
   CONTACT: 'hasContact',
   ADD_FUNDS: 'addFunds',
+  ADD_FUNDS_FROM_COLLECTIVE: 'addFundsFromCollective',
```
Comment (Betree) [inline components/collective-navbar/menu.js:15]: The same feedback about the variable name applies here too

File: components/collective-navbar/index.js [hunk 3]
```diff
@@ -258,6 +258,7 @@ const getDefaultCallsToActions = (collective, sections, isAdmin, isHostAdmin, Lo
     hasRequestGrant:
       (isFund || get(settings, 'fundingRequest') === true) && expenseSubmissionAllowed(collective, LoggedInUser),
     addFunds: isHostAdmin,
+    addFundsFromCollective: isAdmin && collective.type === CollectiveType.EVENT,
```
Comment (Betree) [inline components/collective-navbar/index.js:261]: The same feedback about the variable name applies here too

File: components/collective-navbar/ActionsMenu.js [hunk 4]
```diff
@@ -386,6 +411,8 @@ CollectiveNavbarActionsMenu.propTypes = {
     hasContribute: PropTypes.bool,
     /** Add funds to a collective */
     addFunds: PropTypes.bool,
+    /** Add funds from the parent Collective to an event */
+    addFundsFromCollective: PropTypes.bool,
```
Comment (Betree) [inline components/collective-navbar/ActionsMenu.js:415]: The same feedback about the variable name applies here too

**Review by Betree — APPROVED** [review]
Looks good to me

**Comment by SudharakaP** [comment]
> Looks good to me

Thanks for the review. 👍🏽

**Comment by znarf** [comment]
@SudharakaP I have some concerns about this issue. Please don't deploy it or put it behind a feature flag, deactivated by default.

**Pull request merged**